File: app/src/main/java/com/example/kotlinworkspace/hackerrank/Interview.java
package com.example.kotlinworkspace.hackerrank;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;
import java.util.stream.IntStream;

/**
 * Created by Nalan Ulusoy on 26,Ekim,2022
 */
public class Interview {
    Long i = 1L ;
    public static void main(String[] args) {
        int sum = 0;
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for (int i = n; n >= 3; i--) {
            if (n % 3 == 0 || n % 5 == 0) {
                sum += sum;
            }
        }

        System.out.println(sum);
    }

}

File: app/src/main/java/com/example/kotlinworkspace/hackerrank/Interview.kt
package com.example.kotlinworkspace.hackerrank

import java.util.*


/**
 * Created by Nalan Ulusoy on 26,Ekim,2022
 */


fun main(args: Array<String>) {
    var sum = 0
    val `in` = Scanner(System.`in`)
    val n = `in`.nextInt()
    var i = n
    while (n >= 3) {
        if (n % 3 == 0 || n % 5 == 0) {
            sum += sum
        }
        i--
    }
    println(sum)
}
fun sumSquare(n: Int): Int {
    var count = 0;
    var i: Long = 1
    while (i * i <= n) {
        var j: Long = 1
        while (j * j <= n) {
            if (i * i + j * j == n.toLong()) {
                count += 1

            }
            j++
        }
        i++
    }
    return count
}